draft-ietf-rtcweb-jsep.xml

```diff
@@ -1661,11 +1744,11 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
           or not depends on the bundle policy.
           </t>
 
-          <t>The next step is to generate session-level lip sync groups as
-          defined in <xref target="RFC5888"/>, Section 7.  For each MediaStream
-          with more than one MediaStreamTrack, a group of type "LS" MUST be
-          added that contains the mid values for each MediaStreamTrack in that
-          MediaStream. </t>
+          <t>The next step is to generate session-level lip sync
+          groups as defined in <xref target="RFC5888"/>, Section
+          7. For each MediaStream referenced by more than one
+          RtpTransceiver, a group of type "LS" MUST be added that
```

Comment:
I think it's easier to think of the LS group as one of transceivers, since they are identified in the SDP by m-line.  But if they are tied together by addTrack, which returns an RtpSender, then the transceiver is implicit.